templates/activemq.xml.erb

```diff
@@ -116,7 +124,7 @@
 
         <% if @mq_cluster_type == "nocluster" -%>
         <persistenceAdapter>
-             <kahaDB directory="/var/activemq/kahadb"/>
+             <kahaDB directory="/var/activemq/kahadb" preallocationStrategy="zeros"/>
```

Comment:
For background on this change see: http://blog.christianposta.com/activemq/speeding-up-activemq-persistent-messaging-performance-by-25x/